docs/source/DeveloperGuide/Systems/RequestManagement/index.rst

```diff
@@ -112,6 +112,11 @@ Once the request is ready, you can insert it to the `ReqDB`::
   >>> rc = ReqClient() # # create client
   >>> rc.putRequest( request ) # # put request to ReqDB
 
+.. DANGER::
```

Comment:
```suggestion
.. warning::
```
warning gives it a nice read box: e.g. https://dirac.readthedocs.io/en/latest/CodeDocumentation/Core/Utilities/Adler.html

I think this should also go to the Operation.addFile function and maybe the File module or constructor